### DISCUSSION TITLE
How many shell processes and what kind the server is using

### DISCUSSION THREAD

**Discussion by jcubic** [apollographql/apollo-server#7734] [Q&A] [answered]
So I have two APIs on my shared hosting and it seems when I make one request to the API it reaches the limit of processes that I have on my hosting. Yesterday my server was down because there were 40 processes running at the same time. The static sites were working but the one that was written in PHP gave a 503 error from the server.

The problem is that those apps are not used by anyone (at least not by real users).

So the question is what kind of processes the server is using, how many of them are created, and for what purpose?

**Comment by trevor-scheer** [comment, marked as answer]
Apollo Server runs on 1 node process and doesn't spawn any others, and it's for the purpose of running your GraphQL API. I think your problem lies elsewhere. Have you investigated what those other processes are?

**Reply by jcubic** [reply]
Thanks for the answer, maybe the problem is with express.js
